src/pages/Domains/index.tsx

```diff
@@ -57,7 +56,7 @@ interface DomainsProps {
 }
 
 function Domains({ address }: DomainsProps) {
-  const { emptyDomainArray, domainArray, loading } = useDomainsData(address);
+  const { data, loading } = useGetENSDomainsByAddress(address);
```

Comment:
Think about renaming `data` to something which it represents; in this case it could be **domains**, **domainsData**, **domainsList**, etc.